docs-source/content/samples/simple/domains/model-in-image/update4.md

```diff
@@ -54,6 +57,31 @@ Here are the steps:
      - To make it obvious which ConfigMap belong to which domains.
      - To make it easier to clean up a domain. Typical cleanup scripts use the `weblogic.domainUID` label as a convenience for finding all resources associated with a domain.
 
+1. Update the data source secret that you created in the Update 1 use case with a correct URL, a correct password, as well as with an increased maximum pool capacity:
+
+   ```
+   $ kubectl -n sample-domain1-ns delete secret sample-domain1-datasource-secret
+   $ kubectl -n sample-domain1-ns create secret generic \
+      sample-domain1-datasource-secret \
+      --from-literal='user=sys as sysdba' \
+      --from-literal='password=Oradoc_db1' \
+      --from-literal='max-capacity=10' \
+      --from-literal='url=jdbc:oracle:thin:@oracle-db.default.svc.cluster.local:1521/devpdb.k8s'
+   $ kubectl -n sample-domain1-ns label  secret \
+      sample-domain1-datasource-secret \
+      weblogic.domainUID=sample-domain1
+   ```
+
+1. Optionally start the database.
+
+    - If the database is running, then the sample application that we will run at the end of this step will verify that your updates to the data source secret took effect.
+
+    - If you are taking the `JRF` path through the sample, then the database will already be running. 
+
+    - If you are taking the `WLS` path through the sample, then you can deploy the database by:
+      - Following the the first step in [Set up and initialize an infrastructure database]({{< relref "/samples/simple/domains/model-in-image/prerequisites.md#set-up-and-initialize-an-infrastructure-database" >}}). This step is titled `Ensure that you have access to the database image, and then create a deployment`.
+      - You can skip the remaining steps (they are only needed for `JRF`).
+
```

Comment:
the kubectl edit command could use a little more help - provide the following snippet (substitute JRF for WLS for the JRF path):
```
  configuration:
    model:
      configMap: sample-domain1-wdt-config-map
      domainType: WLS
      onlineUpdate:
        enabled: true
      runtimeEncryptionSecret: sample-domain1-runtime-encryption-secret
```

Reply:
Also, the patch command should use 'true' not "true".